### PR TITLE
add limit + style drop down list

### PR DESCRIPTION
**Pull request by Ebazhanov**
image

### CODE REVIEW

File: src/components/Autocomplete/styled.ts [hunk 1]
```diff
@@ -19,4 +19,10 @@ const StyledIconButton = styled.div`
     }
 `;
 
-export { StyledChipContainer, StyledIconButton, StyledTextInput };
+const StyledPaper = styled(Paper)`
+    .MuiAutocomplete-option:hover {
+        background-color: #edf6f8;
```
Comment (stanimir-stoychev) [inline src/components/Autocomplete/styled.ts:24]: Don't hardcode colors

File: src/components/Autocomplete/Autocomplete.tsx [hunk 2]
```diff
@@ -21,6 +21,8 @@ function Autocomplete(props: AutocompletePropsTypes) {
                 {...rest}
                 renderTags={() => null}
                 renderInput={(params) => <StyledTextInput {...params} variant="outlined" placeholder={placeholder} />}
+                renderOption={(option: OptionType) => <Typography>{option.title}</Typography>}
+                PaperComponent={({ children }) => <StyledPaper>{children}</StyledPaper>}
```
Comment (stanimir-stoychev) [inline src/components/Autocomplete/Autocomplete.tsx:25]: There is no way for me to pass anything here as a dev... Add these props to the component's props as well for flexibility

File: src/components/stories/Autocomplete.stories.tsx [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import React, { useState } from 'react';
+import { createFilterOptions } from '@material-ui/lab';
```
Comment (stanimir-stoychev) [inline src/components/stories/Autocomplete.stories.tsx:2]: Maybe you can export this function as well then, it seems useful and once we publish the DS it won't be available unless you export it explicitly

**Review by stanimir-stoychev — CHANGES_REQUESTED** [review]

File: src/components/Autocomplete/Autocomplete.tsx (hunk 2 above)
Comment (stanimir-stoychev) [reply src/components/Autocomplete/Autocomplete.tsx:25]: You still didn't add them, though... There's no way for the developer to provide something else for the PaperComponent prop and the renderOption one too...

File: src/style/theme/hover.ts [hunk 4]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable @typescript-eslint/prefer-as-const */
+const hover = {
+    mainHover: {
+        bgColor: '#EDF6F8' as '#EDF6F8',
```
Comment (stanimir-stoychev) [inline src/style/theme/hover.ts:4]: You just extended the theme? Why? Isn't the color there already? If not then maybe we shouldn't use it... You need to collaborate that with Rafa and try not to change the theme whenever you get a value that doesn't make sense

File: src/style/theme/index.ts [hunk 5]
```diff
@@ -7,6 +8,7 @@ import typography from './typography';
 
 export default {
     breakpoints,
+    hover,
```
Comment (stanimir-stoychev) [inline src/style/theme/index.ts:11]: Hover? What's the idea of this? Are all of our components going to hover with the same color? What's the point of having the palettes if we're just going to do this instead?

**Review by stanimir-stoychev — CHANGES_REQUESTED** [review]

File: src/components/Autocomplete/Autocomplete.tsx [hunk 6]
```diff
@@ -46,3 +48,4 @@ function Autocomplete(props: AutocompletePropsTypes) {
 }
```
Comment (dulik06) [inline src/components/Autocomplete/Autocomplete.tsx:48]: The options object should hava a label and value. Please change it here:

` <div key={v.title}>` and `{v.title}` to ` <div key={v.value}>` and `{v.label}`

**Review by dulik06 — CHANGES_REQUESTED** [review]

File: src/components/Autocomplete/types.ts [hunk 7]
```diff
@@ -1,10 +1,9 @@
-/* eslint-disable @typescript-eslint/ban-types */
 import * as React from 'react';
 import { AutocompleteProps } from '@material-ui/lab';
 import { WithClassAndId, WithCssGeneratorFunction } from '../../core';
 import { LabelWithProps } from '../shared';
 
-export type OptionType = { title: string; value: string };
+export type OptionType = { lable: string; value: string };
```
Comment (stanimir-stoychev) [inline src/components/Autocomplete/types.ts:6]: Why `table`?

File: src/components/Autocomplete/styled.ts [hunk 8]
```diff
@@ -1,9 +1,10 @@
 import styled from 'styled-components';
-import { TextField } from '@material-ui/core';
+import { Paper, TextField } from '@material-ui/core';
 
 const StyledChipContainer = styled.div`
     display: flex;
     flex-wrap: wrap;
+    padding: 10px;
```
Comment (stanimir-stoychev) [inline src/components/Autocomplete/styled.ts:7]: Use the `theme`

**Review by stanimir-stoychev — CHANGES_REQUESTED** [review]

File: src/components/Autocomplete/styled.ts [hunk 9]
```diff
@@ -1,9 +1,10 @@
 import styled from 'styled-components';
-import { TextField } from '@material-ui/core';
+import { Paper, TextField } from '@material-ui/core';
 
 const StyledChipContainer = styled.div`
     display: flex;
     flex-wrap: wrap;
+    padding: ${({ theme: { spacing } }) => spacing(2)}; ;
```
Comment (stanimir-stoychev) [inline src/components/Autocomplete/styled.ts:7]:
```suggestion
    padding: ${({ theme: { spacing } }) => spacing(2)};
```

**Review by stanimir-stoychev — DISMISSED** [review]
LGTM

**Review by stanimir-stoychev — APPROVED** [review]
LGTM

**Pull request merged**